pkg/apis/externaldns/binders_test.go

```diff
@@ -179,6 +182,8 @@ func TestCobraRegexValueSetStringType(t *testing.T) {
 
 func TestCobraRegexpVarDefaultAndInvalidValue(t *testing.T) {
 	cmd := &cobra.Command{Use: "test"}
+	cmd.SetErr(io.Discard)
+	cmd.SetOut(io.Discard)
```

Comment:
```suggestion
	cmd := &cobra.Command{Use: "test", SilenceErrors: true, SilenceUsage: true}
```
Could also be moved to a function.